main.go

```diff
@@ -54,14 +54,19 @@ func init() {
 	// +kubebuilder:scaffold:scheme
 }
 
-func newManager() (ctrl.Manager, error) {
+func newManager() (ctrl.Manager, int, error) {
 	var configFile string
 
+	var maxConcurrentReconciles int
+
 	flag.StringVar(&configFile, "config", "",
 		"The controller will load its initial configuration from this file. "+
 			"Omit this flag to use the default configuration values. "+
 			"Command-line flags override configuration from this file.")
 
+	flag.IntVar(&maxConcurrentReconciles, "maxConcurrentReconciles", 1,
```

Comment:
We need all config parameters from the config map above, IOW no CLI arguments as per [this](https://book.kubebuilder.io/component-config-tutorial/tutorial.html)

Reply:
I see. I'll make the change.